main_app/views.py

```diff
@@ -71,7 +67,7 @@ def logout(req):
 @login_required(login_url='/login')
 def account(req):
   user = req.user
-  carts = Cart.objects.filter(user_id=req.user.id).exclude(status=Cart.Status.ABANDONED).order_by('-ordered_at')
+  carts = Cart.objects.filter(user_id=req.user.id).exclude(Q(status=1) | Q(status=-1)).order_by('-ordered_at')
```

Comment:
Is it possible to use `Status.CURRENT` and `Status.ABANDONED` instead of the numbers directly?